test/token/erc721/ImmutableERC721HybridPermissionedMintable.test.ts

```diff
@@ -163,6 +163,72 @@ describe("ImmutableERC721", function () {
       );
     });
 
+    it("Should not allow owner or approved to burn a token when specifying the incorrect owner", async function() {
+      await expect(
+        erc721.connect(user).safeBurn(owner.address, 5)
+      ).to.be.revertedWith('IImmutableERC721MismatchedTokenOwner');
```

Comment:
lets add the `withArgs` after the `revertedWith`

Reply:
Done.